Disher/View Controllers/RecipesViewController.m

```diff
@@ -25,17 +26,10 @@ - (void)viewDidLoad {
     
     self.tableView.delegate = self;
     self.tableView.dataSource = self;
-    // Uncomment the following line to preserve selection between presentations.
-    // self.clearsSelectionOnViewWillAppear = NO;
-    
-    // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
-    // self.navigationItem.rightBarButtonItem = self.editButtonItem;
-    [self queryMealDB:@"Chicken" completionHandler:^(NSArray *meals) {
-        self.mealDBresults = meals;
+    //Update global results arrays
+    [self queryAPIs:@"banana" completionHandler:^() {
```

Comment:
suggestion: start designing your code in a way that's later easier to customize.

for example, instead of a literal string, crate a constant, which can later be changed for different tests, and eventually be a variable populated through a search bar, etc.